docs/REST API/rest-api.md

```diff
@@ -487,17 +487,17 @@ for pagination. For a comprehensive list of activity types and detailed informat
     {
       "created_at": "2023-07-27T14:35:08Z",
       "id": 25,
-      "actor_full_name": "Anna Chao",
-      "actor_id": 3,
+      "actor_full_name": "Fleet",
       "actor_gravatar": "",
       "actor_email": "",
-      "type": "uninstalled_software",
+      "type": "installed_software",
+      "fleet_initiated": "true", // If Fleet installed software via policy automation
```

Comment:
If we think we need this description, I'd put it in a note elsewhere instead of a comment here; would maybe start without it since this definition could eventually expand 
```suggestion
      "fleet_initiated": "true",
```